docs/mobile/game-session/game_session_db_design.md

```diff
@@ -0,0 +1,53 @@
+# DynamoDB Design
+
+## Entities
+
+### Config
+
+This entity contains the global settings and configurable parts of the application. The list of items included here is as follow
+
+- Points granted to each team estimated the right popular trick answer
+- Points granted per to teams guessed the right answer
+
+### GameSession
+
+The higher level entity contains all information related to a game from start to finish.
+
+- Game: A game object that comes from Aurora. It's a snapshot of the data when the game starts
+- Teams: An array of team entities that involved in the game
+- StartedDateTimeUTC: Date time of when the game started in UTC
+- MultiDeviceMode: Boolean indicating if it's a multi-device with facilitator or not
+
+### Team
+
+- TeamNo / TeamId: Numeric unique number assigned to the team
+- Team name: Name chosen to show fo the team. This helps for future substituting `team 1` string with chosen team names
+- Question: The chosen question for the team
+- GameCode: A 4-digit code to enter to join the team
+
+### TeamMember
+
+- TeamId: The teamId this member is a member of
+- DeviceId: Unique number associated to the device
+- IsFacilitator: A boolean indicating if this device is the facilitator in multi-device mode
+
+### Question
```

Comment:
These Question entities would also be in the GameSession.Game. Is this section describing the field types for Questions, or are we thinking about pulling them out of the Game object? If so, does the data ever change?

Reply:
Yes, actually I missed to layout the `Game` entity here. I'm listing all entities that are involved on the mobile side and then their relationships. Then we can decide if questions will be `GameSession.Game.Questions` or be accessible as a separate entity.